src/server/ns_turn_server.c

```diff
@@ -4681,80 +4681,79 @@ static void peer_input_handler(ioa_socket_handle s, int event_type, ioa_net_data
 
   int ilen =
       min((int)ioa_network_buffer_get_size(in_buffer->nbh), (int)(ioa_network_buffer_get_capacity_udp() - offset));
+  if (ilen < 0)
+    return;
 
-  if (ilen >= 0) {
-    ++(ss->peer_received_packets);
-    ss->peer_received_bytes += ilen;
-    turn_report_session_usage(ss, 0);
+  ++(ss->peer_received_packets);
+  ss->peer_received_bytes += ilen;
+  turn_report_session_usage(ss, 0);
 
-    allocation *a = get_allocation_ss(ss);
-    if (is_allocation_valid(a)) {
+  allocation *a = get_allocation_ss(ss);
+  if (!is_allocation_valid(a))
+    return;
 
-      uint16_t chnum = 0;
+  uint16_t chnum = 0;
 
-      ioa_network_buffer_handle nbh = NULL;
+  ioa_network_buffer_handle nbh = NULL;
 
-      turn_permission_info *tinfo = allocation_get_permission(a, &(in_buffer->src_addr));
-      if (tinfo) {
-        chnum = get_turn_channel_number(tinfo, &(in_buffer->src_addr));
-      } else if (!(server->server_relay)) {
-        return;
-      }
+  turn_permission_info *tinfo = allocation_get_permission(a, &(in_buffer->src_addr));
+  if (tinfo) {
+    chnum = get_turn_channel_number(tinfo, &(in_buffer->src_addr));
+  } else if (!(server->server_relay)) {
+    return;
+  }
 
-      if (chnum) {
+  if (chnum) {
 
-        size_t len = (size_t)(ilen);
+    size_t len = (size_t)(ilen);
 
-        nbh = in_buffer->nbh;
+    nbh = in_buffer->nbh;
```

Comment:
can the declaration and definition of `nbh` happen on the same line?

Reply:
It is possible after C99, but not before.
CMAKE is must supports C11, but MAKEFILE does not necessarily.

Reply:
The reason why it is not defined on the same line is because this variable is used in multiple code blocks

Reply:
I don't see any value in bothering with c99